packages/providers/src/fuel-graphql-request.ts

```diff
@@ -0,0 +1,48 @@
+import type { DocumentNode } from 'graphql';
```

Comment:
For easy reference in the future.

```suggestion
/* 

  Replaces `graphql-request` and `graphql-tag` dependencies with a simple
  `fetch` implementation, reducing the final bundle size by 15.68 kB.

  More:
    https://github.com/FuelLabs/fuels-ts/discussions/1592#discussioncomment-8003515

*/

import type { DocumentNode } from 'graphql';
```